pkg/node/factories.go

```diff
@@ -139,3 +129,25 @@ func NewStandardPublishersFactory() PublishersFactory {
 			return provider.NewConfiguredProvider(pr, nodeConfig.DisabledFeatures.Publishers), err
 		})
 }
+
+func NewStandardAuthenticatorsFactory() AuthenticatorsFactory {
```

Comment:
Can we pretty please not use this pattern for this. I would find it way simple to provide the requester node with a map of method names to Authenticators. :pray: begging you.

Reply:
Very happy to do the refactor work if we have a better way to do all of them, for sure.